File: python/7 kyu/415.py
# https://www.codewars.com/kata/5959ec605595565f5c00002b/train/python

# Passed

def reverse_bits(n):
    n_binary = bin(n).split('b')[-1]
    result = int(n_binary[::-1], 2)
    return result

output = reverse_bits(417)
print(output)

File: nodejs/7 kyu/415.js
// https://www.codewars.com/kata/588422ba4e8efb583d00007d/train/javascript

// Passed

function lateRide(n) {
  let hours = Math.floor(n / 60);
  let minutes = n - hours * 60;
  let getNumberDigitsSum = (num) =>
    Array.from(String(num))
      .map((strDigit) => Number(strDigit))
      .reduce((prev, curr) => prev + curr, 0);

  let result = getNumberDigitsSum(hours) + getNumberDigitsSum(minutes);
  return result;
}

const output = lateRide(1439);
console.log(output);